src/CrowdedMod/CrowdedMod.csproj

```diff
@@ -11,11 +11,11 @@
     </PropertyGroup>
     <PropertyGroup>
         <GamePlatform Condition="'$(GamePlatform)' == ''">Steam</GamePlatform>
-        <GameVersion>2023.3.28</GameVersion>
+        <GameVersion>2023.7.11</GameVersion>
```

Comment:
If you want to bump among us version for no reason, you should test it on v2023.3.28 because current CrowdedMod version supports v2023.3.28-v2023.7.11

Reply:
reverted the change